public/wordList.js

```diff
@@ -0,0 +1,8909 @@
+    'wellington',
+    'wellness',
+    'wells',
+    'welsh',
+    'wendy',
+    'went',
+    'were',
+    'wesley',
+    'west',
+    'western',
+    'westminster',
+    'whale',
+    'what',
+    'whatever',
+    'whats',
+    'wheat',
+    'wheel',
+    'wheels',
+    'when',
+    'whenever',
+    'where',
+    'whereas',
+    'wherever',
+    'whether',
+    'which',
+    'while',
+    'whilst',
+    'white',
+    'whole',
+    'wholesale',
+    'whom',
+    'whore',
+    'whose',
+    'wichita',
+    'wicked',
+    'wide',
+    'widely',
+    'wider',
+    'widescreen',
+    'widespread',
+    'width',
+    'wife',
+    'wifi',
+    'wiki',
+    'wikipedia',
+    'wild',
+    'wilderness',
+    'wildlife',
+    'wiley',
+    'will',
+    'william',
+    'williams',
+    'willing',
+    'willow',
+    'wilson',
+    'wind',
+    'window',
+    'windows',
+    'winds',
+    'windsor',
+    'wine',
+    'wines',
+    'wing',
+    'wings',
+    'winner',
+    'winners',
+    'winning',
+    'wins',
+    'winston',
+    'winter',
+    'wire',
+    'wired',
+    'wireless',
+    'wires',
+    'wiring',
+    'wisconsin',
+    'wisdom',
+    'wise',
+    'wish',
+    'wishes',
+    'wishlist',
+    'witch',
+    'with',
+    'withdrawal',
+    'within',
+    'without',
+    'witness',
+    'witnesses',
+    'wives',
+    'wizard',
+    'wolf',
+    'woman',
+    'women',
+    'womens',
+    'wonder',
+    'wonderful',
+    'wondering',
+    'wood',
+    'wooden',
+    'woods',
+    'wool',
+    'worcester',
+    'word',
+    'wordpress',
+    'words',
+    'work',
+    'worked',
+    'worker',
+    'workers',
+    'workflow',
+    'workforce',
+    'working',
+    'workout',
+    'workplace',
+    'works',
+    'workshop',
+    'workshops',
+    'workstation',
+    'world',
+    'worldcat',
+    'worlds',
+    'worldsex',
+    'worldwide',
+    'worm',
+    'worn',
+    'worried',
+    'worry',
+    'worse',
+    'worship',
+    'worst',
+    'worth',
+    'worthy',
+    'would',
+    'wound',
+    'wrap',
+    'wrapped',
+    'wrapping',
+    'wrestling',
+    'wright',
+    'wrist',
+    'write',
+    'writer',
+    'writers',
+    'writes',
+    'writing',
+    'writings',
+    'written',
+    'wrong',
+    'wrote',
+    'wyoming',
+    'xanax',
+    'xbox',
+    'xerox',
+    'xhtml',
+    'xnxx',
+    'yacht',
+    'yahoo',
+    'yale',
+    'yamaha',
+    'yang',
+    'yard',
+    'yards',
+    'yarn',
+    'yeah',
+    'year',
+    'yearly',
+    'years',
+    'yeast',
+    'yellow',
+    'yemen',
+    'yesterday',
+    'yield',
+    'yields',
+    'yoga',
+    'york',
+    'yorkshire',
+    'young',
+    'younger',
+    'your',
+    'yours',
+    'yourself',
+    'youth',
+    'yugoslavia',
+    'yukon',
+    'zambia',
+    'zdnet',
+    'zealand',
+    'zero',
+    'zimbabwe',
+    'zinc',
+    'zoloft',
+    'zone',
+    'zones',
+    'zoning',
+    'zoom',
+    'zoophilia',
+    'zope',
+    'zshops'
+]
+
+function returnWord() {
```

Comment:
the function is not exported, cant be used by others. can you put this function on the top?